lib/rules/disallow-space-after-keywords.js

```diff
@@ -31,7 +31,7 @@ module.exports.prototype = {
         file.iterateTokensByType('Keyword', function(token, i, tokens) {
             if (keywordIndex[token.value]) {
                 var nextToken = tokens[i + 1];
-                if (nextToken && nextToken.range[0] !== token.range[1]) {
```

Comment:
Here I can't imagine a situation where nextToken for Keyword wouldn't exist. Probably I'm wrong.